src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/QuizAssignmentManagerTest.java

```diff
@@ -63,7 +64,7 @@ public void setUp() throws ContentManagerException, SegueDatabaseException {
     replay(properties, emailService, quizAssignmentPersistenceManager);
   }
 
-  @Before
+  @BeforeEach
```

Comment:
I know you're only updating this, but having multiple `@BeforeEach` in the same class can be dangerous, as the execution order is not guaranteed. In this particular case, it's fine, because both of them seem to be independent, but I would personally get rid of this method and simply make its contents part of the `setUp` method.

Reply:
We did deal with an instance of this yesterday that was causing a handful of test failures. Didn't realise there was another, will have to take a look...